pkg/kubelet/kubelet.go

```diff
@@ -375,6 +374,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	crOptions *config.ContainerRuntimeOptions,
 	containerRuntime string,
 	hostnameOverride string,
+	nodeName types.NodeName,
```

Comment:
```suggestion
	hostname string,
	hostnameOverridden string,
	nodeName types.NodeName,
```

Reply:
Thank you very much, very good advice, so that hostname, nodename, hostnameOverridden can avoid double compute, has been completed